### PR TITLE
[Allwinner,Generic,Rockchip] linux: Update to 5.10.35

### PR DESCRIPTION
**Pull request by heitbaum**
https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.10.31
https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.10.32
https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.10.33
https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.10.34
https://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.10.35

5.10.31 - 25 patches
5.10.32 - 103 patches
5.10.33 - 36 patches
5.10.34 - 2 minor patches (iwlwifi and addition of Alder Lake device ID)
5.10.35 - 29 patches

log - https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable-rc.git/log/?h=linux-5.10.y

Generic (TGL) - build and run tested 5.10.31,5.10.32,5.10.33,5.10.34,5.10.35 - heitbaum

**Draft patch - but posting it earlier than normal because of allwinner updates**

### CODE REVIEW

**Comment by heitbaum** [comment]
5.10.31 Build tested on all of:
5.10.32 Build tested on all of (including #5330):
```
PROJECT=Allwinner ARCH=arm DEVICE=H6 UBOOT_SYSTEM=tanix-tx6 s/build linux
PROJECT=Rockchip ARCH=arm DEVICE=RK3288 s/build linux
PROJECT=Rockchip ARCH=arm DEVICE=RK3328 s/build linux
PROJECT=Rockchip ARCH=arm DEVICE=RK3399 s/build linux
PROJECT=NXP ARCH=arm DEVICE=iMX6 s/build linux
PROJECT=NXP ARCH=arm DEVICE=iMX8 s/build linux
PROJECT=Qualcomm ARCH=arm DEVICE=Dragonboard s/build linux
PROJECT=Samsung ARCH=arm DEVICE=Exynos s/build linux
PROJECT=Generic ARCH=x86_64 s/build linux
```

File: projects/Allwinner/patches/linux/0061-arm64-dts-allwinner-Fix-SD-card-CD-GPIO-for-SOPine-s.patch [hunk 1]
```diff
@@ -38,18 +38,6 @@ index 437ffe3628a5..e79ce49e7e6a 100644
 +&mmc0 {
 +	cd-gpios = <&pio 5 6 GPIO_ACTIVE_LOW>; /* PF6 push-push switch */
 +};
-diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-sopine.dtsi
```
Comment (jernejsk) [inline projects/Allwinner/patches/linux/0061-arm64-dts-allwinner-Fix-SD-card-CD-GPIO-for-SOPine-s.patch:41]: You should remove whole patch. Partial removal probably works but causes two mmc0 nodes.

Comment (heitbaum) [reply projects/Allwinner/patches/linux/0061-arm64-dts-allwinner-Fix-SD-card-CD-GPIO-for-SOPine-s.patch:41]: Done. - full,patch dropped.

**Comment by heitbaum** [comment]
Revert included in 5.10.33-rc1
- arm64: dts: allwinner: Revert SD card CD GPIO for Pine64-LTS
- https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable-rc.git/commit/?h=linux-5.10.y&id=25412f1f0488df76ff1197f66f1877b3e6d55ee0

**Comment by CvH** [comment]
@HiassofT @jernejsk @knaerzche good to go ?

**Comment by jernejsk** [comment]
last time I tested (.33 I think) everything was ok, so gtg (lastest versions don't change anything big)

**Comment by knaerzche** [comment]
GTG for RK

**Pull request merged**